server/datastore/mysql/migrations/tables/20231212095734_AddSoftwareTitlesHostsCountsTable.go

```diff
@@ -0,0 +1,35 @@
+package tables
+
+import (
+	"database/sql"
+
+	"github.com/pkg/errors"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20231212095734, Down_20231212095734)
+}
+
+func Up_20231212095734(tx *sql.Tx) error {
+	softwareTitlesHostCountsTable := `
+    CREATE TABLE IF NOT EXISTS software_titles_host_counts (
+      software_title_id int(10) unsigned NOT NULL,
+      hosts_count bigint(20) unsigned NOT NULL,
```

Comment:
sanity checking, in `software_host_counts` the id reference is `bigint(20)` and the count is `int(10)`. Seems like it was meant to be the other way around? I can switch it back just lmk.

https://github.com/fleetdm/fleet/blob/87519239a20f1f006e6d3d0390bf6a01aebf5ccc/server/datastore/mysql/schema.sql#L1282-L1283

Reply:
`software_host_counts` has `software_id` as a BIGINT because the `software` table's `id` is BIGINT (we probably feared running into the max of the 32-bit int at some point). The count being `INT UNSIGNED` is fine, it's still 4 billion+ hosts.

For `software_titles_host_counts`, `software_title_id` is ok at `INT UNSIGNED` as it refers to `software_titles` primary key, but I'd leave the count column `INT UNSIGNED` too, no need for BIGINT here IMO.

Reply:
that makes a lot of sense. Thank you!